packages/component-model-manuscript/entities/manuscript/index.js

```diff
@@ -204,12 +203,12 @@ class Manuscript extends BaseModel {
               this.updated
             }`,
           )
-          // Temporarily commented out see #1162
-          // throw integrityError(
-          //   'updated',
-          //   this.updated,
-          //   'is older than the one stored in the database!',
-          // )
+          // Autosave is broken so can fail here
```

Comment:
These lines have been reinstated to highlight that autosave has a race condition with other long running server tasks (such as file upload).